bbs/admin/unix/install.sh

```diff
@@ -326,14 +331,22 @@ then
   fi
 else
   say "User ${WWIV_USER} not found; creating it."
-  useradd -g ${WWIV_GROUP} -c "WWIV BBS Service Account" -d ${WWIV_DIR} \
-	  -s /sbin/nologin -m ${WWIV_USER} >> ${LOGFILE} 2>&1
+  if [[ $OSTYPE =~ "SunOS" ]]
+  then
```

Comment:
@granitepenguin I agree. I just didn't know how to check the architecture on SunOS as the file command isn't compatible with Linux's file command and doesn't have the -e switch.